examples/digital/main.cpp

```diff
@@ -15,6 +15,11 @@ int main()
 
 {
 	M2k *ctx = m2kOpen();
+	if (!ctx) {
```

Comment:
nitpick, and strictly speaking about style;
i still don't know what general style is preferred for this project;
specifically here: is it  `if()` or `if ()` [i.e. `if__space__()`] ?

Reply:
The style is: "if () {"